third.txt

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html>
+<head>
+  <title>jQuery Checkbox state</title>
+  <style>
```

Comment:
Need to remove the style code and add style.css

Reply:
style.css added in seperate file